lib/JMAP/Tester.pm

```diff
@@ -494,6 +494,25 @@ has _jwt_config => (
   init_arg => undef,
 );
 
+sub _now_timestamp {
+  #   0     1     2      3      4     5
+  my ($sec, $min, $hour, $mday, $mon, $year) = gmtime;
+  return sprintf '%04u-%02u-%02uT%02u:%02u:%02uZ',
+    $year + 1900, $mon + 1, $mday,
+    $hour, $min, $sec;
+}
+
+sub _get_jwt_config {
+  my ($self) = @_;
+  return unless my $jwtc = $self->_jwt_config;
+  return $jwtc unless $jwtc->{signingKeyValidUntil};
+  return $jwtc if $jwtc->{signingKeyValidUntil} gt $self->_now_timestamp;
+
+  $self->_update_auth;
+  return unless $jwtc = $self->_jwt_config;
+  return $jwtc;
```

Comment:
Almost feel like we should validate the signingKeyValidUntil again, but it's probably fine if we don't.